src/Telemetry/Admin_Subscriber.php

```diff
@@ -21,6 +21,9 @@ public function register(): void {
 				$this->container->get( Activation_Redirect::class )->trigger();
 			}
 
+			// Load the admin resources.
+			$this->container->get( Admin_Resources::class )->enqueue();
```

Comment:
💡 
Is there a way for us to target the pages we know we need to load on?  The exit interview is pretty straightforward, but on the settings pages we need to include the do_action on might be a bit harder to figure that out.